core/templates/dev/head/domain/summary/ExplorationSummaryBackendApiServiceSpec.js

```diff
@@ -0,0 +1,39 @@
+describe('ExplorationSummaryBackendApiService', function() {
```

Comment:
Add license, Copyright, and file overview.

Reply:
Added license, Copyright, and file overview.